src/components/Audio/Info.js

```diff
@@ -1,5 +1,30 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import * as styles from '@styles/modules/audio.module.scss';
+import { FormattedMessage } from 'react-intl';
+import LocaleContext from '../../context/LocaleContext';
+import { LOCALES } from '../../content/locales';
 
-const Info = () => <h3>Info</h3>;
+const Info = () => {
+  const { locale } = useContext(LocaleContext);
+
+  return (
+    <div className={styles.infoContainer}>
+      <h3
+        className={`${
+          locale === LOCALES.ARABIC ? styles.arabic : styles.english
+        }`}
+      >
+        <FormattedMessage id="info.heading" />
+      </h3>
+      <p
+        className={`${
+          locale === LOCALES.ARABIC ? styles.arabic : styles.english
```

Comment:
One minor suggestion as the application grows–

You may want to consider implementing this conditional className at a higher level in the application (possibly in `pages/index.js`) so you aren't needing to implement on every individual text field. 

The trade-off being it would break the self-contained nature of your current components.

Reply:
Noted. We plan on doing just that.